suite/suite.go

```diff
@@ -55,6 +55,21 @@ func (suite *Suite) Assert() *assert.Assertions {
 	return suite.Assertions
 }
 
+// Run is a helper method to do a simple subtest. It sets `T()` to the
+// `*testing.T` of the subtest.
+func (suite *Suite) Run(name string, f func()) {
+	oldT := suite.T()
+	suite.T().Run(name, func(t *testing.T) {
+		suite.SetT(t)
+
+		defer func() {
+			suite.SetT(oldT)
+		}()
```

Comment:
Could this result in race conditions if somebody called `suite.T().Parallel()` within the subtest?